### DISCUSSION TITLE
Add donation info to this project's github page

### DISCUSSION THREAD

**Discussion by aha999** [searx/searx#2900] [Ideas]
**Is your feature request related to a problem? Please describe.**
I think that SearX is awesome, but there is no donation info where users can help sponsor/donate to this project.

**Describe the solution you'd like**
Add donation info to SearX github page.

**Describe alternatives you've considered**
If you actually don't yet have donation plaforms, i could recommend LiberaPay,  OpenCollective, Ko-fi.

**Additional context**
IT would be reallz nice if users could help out and donate to such a great project to maybe help it grow or to show appreciation for the hard work, that's why donation info should be very visible.

**Comment by aha999** [comment]
I see that donation info is in "badges", it's not clear... I think that donations should be more exposed.

 also see that donation info is here with more methods of donations, why not add them to the Readme ? https://searx.me/static/donate.html

**Comment by unixfox** [comment]
@asciimoo

**Comment by asciimoo** [comment]
Thank you, I'm going to update the readme. Note that we have an OpenCollective account: https://opencollective.com/searx . Any kind of support is appreciated.
